diff --git a/coq-graph-theory.opam b/coq-graph-theory.opam
index 2a9ea1d..8487df7 100644
--- a/coq-graph-theory.opam
+++ b/coq-graph-theory.opam
@@ -20,15 +20,18 @@ from the study of relation algebra within the ERC CoVeCe project
 (e.g., soundness and completeness of an axiomatization of graph
 isomorphism)."""
 
-build: [make "-j%{jobs}%" ]
+build: [
+  ["sh" "-exc" "cat _CoqProject.wagner >>_CoqProject"] {coq-fourcolor:installed}
+  [make "-j%{jobs}%" ]
+]
 install: [make "install"]
 depends: [
   "coq" {(>= "8.12" & < "8.14~") | (= "dev")}
   "coq-mathcomp-algebra" {(>= "1.12" & < "1.13~") | (= "dev")}
   "coq-mathcomp-finmap" 
   "coq-hierarchy-builder" { (>= "1.1.0") }
-  "coq-fourcolor" 
 ]
+depopts: ["coq-fourcolor"]
 
 tags: [
   "category:Computer Science/Graph Theory"
